examples/src/main/python/als.py

```diff
@@ -62,10 +62,10 @@ def update(i, mat, ratings):
       example. Please use pyspark.ml.recommendation.ALS for more
       conventional use.""", file=sys.stderr)
 
-    spark = SparkSession\
-        .builder\
-        .appName("PythonALS")\
-        .getOrCreate()
+    spark = (SparkSession
```

Comment:
I have not changed all this initilization lines, since they do not appear most of the time in the documentation